server/sonar-server/src/main/java/org/sonar/server/component/ws/ProjectMeasuresQueryValidator.java

```diff
@@ -45,6 +45,9 @@ public void validate(DbSession dbSession, ProjectMeasuresQuery query) {
       return;
     }
     List<MetricDto> dbMetrics = dbClient.metricDao().selectByKeys(dbSession, new ArrayList<>(metricKeys));
+
+    // TODO validate sort value : either a metric key, or 'name'
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135)